src/ui/Typeahead.tsx

```diff
@@ -97,6 +97,7 @@ const TypeaheadInner = <T extends TypeaheadModel>(
 
         .dropdown-item {
           border-bottom: 1px solid ${({ theme }) => theme.colors.grey1};
+          text-wrap: auto;
```

Comment:
Did you check the other dropdowns? 
To see if they still look ok?

Reply:
hmm no it shouldn't affect too much as a standalone property but depends of some of the layouts, will see if I spot something weird